package.json

```diff
@@ -27,12 +27,12 @@
     "@typescript-eslint/eslint-plugin": "^6.2.1",
     "@typescript-eslint/parser": "^6.2.1",
     "eslint": "^8.46.0",
-    "eslint-config-prettier": "^8.9.0",
+    "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-jest": "^27.2.3",
     "husky": "^4.2.5",
     "jest": "^29.6.1",
     "jest-extended": "^4.0.0",
-    "lerna": "^6.0.0",
+    "lerna": "^7.1.4",
```

Comment:
Needed to bump this due to a compatibility issue between Lerna 6 and Prettier 3. Here was the fix in Lerna: https://github.com/lerna/lerna/pull/3766/files#diff-f7957c0c5c84c95434097a432a049ba780cdccfd5db0a156de23911e6f5ff776R36